analyses/sample-psi-clustering/03-plot-sbi-with-cluster-mem.R

```diff
@@ -0,0 +1,115 @@
+################################################################################
+# 03-plot-sbi_with_cluster-mem.R
+# script that takes in "splicing_index.total.txt" data file from previous module
+# and intersects with cluster membership info from current module output file 
+# and plots number of samples per cluster stratified by splicing burden
+# 
+# written by Ammar Naqvi, Jo Lynne Rokita, Ryan Corbett
+#
+# usage: Rscript 03-plot-sbi_with_cluster-mem.R
+################################################################################
+library("tidyverse")
+
+#Get `magrittr` pipe
+`%>%` <- dplyr::`%>%`
+
+##directory setup
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+analysis_dir <- file.path(root_dir, "analyses", "clustering_analysis")
+
+plots_dir <- file.path(analysis_dir, "plots")
+input_dir <- file.path(analysis_dir, "input")
+output_dir <- file.path(analysis_dir, "output")
```

Comment:
```suggestion
results_dir <- file.path(analysis_dir, "output")
```

based on below naming